HIP-0013.md

```diff
@@ -0,0 +1,130 @@
+# HIP-0013 : Standard for Profile Data via TXT Records 
+
+```
+Number:  HIP-0013
+Title:   Standard for Profile Data via TXT Records 
+Type:    Standards
+Status:  Draft
+Authors: 0xStefan <https://twitter.com/0xStefan>
+Created: 2022-02-22
+```
+
+## Abstract
+
+A proposal to create a standard for the labels of string attributes in TXT records for profile data — following the [RFC 1464](https://datatracker.ietf.org/doc/html/rfc1464) standard. Parts of this standard where inspired by [EIP-634](https://eips.ethereum.org/EIPS/eip-634).
+
+## Motivation
+
+Handshake domains can also be used as usernames. Domains are used to sign in to websites and to identify users in the Handshake ecosystem. Therefore it's useful to be able to attach profile information like an email, phone, avatar or service to a Domain.
+
+This Standard can also be used to provide contact information for domain listings according to [HIP-0010](https://github.com/handshake-org/HIPs/blob/master/HIP-0010.md).
+
+## TXT Record Standard: Profile Data
+
+For the above to work we need standards so any website or app can properly render the TXT records. The records should be flexible, short and easy to understand. It's important to note, that all values should be checked by the application before serving them to the user.
+
+The general syntax for profile data is:
+
+**TXT** `profile <attribute name>=<attribute value>`
+
+There are currently five supported attributes: **[email](#email)**, **[phone](#phone)**, **[avatar](#avatar)** and **[service](#service)**. All attributes are optional but at least one attribute needs to be present, for a Domain to count as a profile.
+
+Although each `<attribute name>=<attribute value>` pair can be provided as individual records, please consider using the **[recommended implementation](#alt)**.
+
+### <a id="email"></a> (1) Profile Attribute: Email
+
+The email attribute is optional. It enables a domain owner to provide an email for the domain. The expected value is an email address.
+
+```
+TXT profile email=<email-address>
+```
+
+#### Valid Email Examples
+
+```
+TXT profile email=hello@niami.io
+TXT profile email=hello@niami
+```
+
+### <a id="phone"></a> (2) Profile Attribute: Phone
+
+The phone attribute is optional. It enables a domain owner to provide a phone number for the domain. The expected value is a phone number as an E.164 string (a number up to fifteen digits in length starting with a '+'). Regex: `^\+[1-9]\d{1,14}$`
+
+```
+TXT profile phone=<phone-number>
+```
+
+#### Valid Phone Examples
+
+```
+TXT profile phone=+15368886253
+TXT profile phone=+47538886253
+```
+
+### <a id="avatar"></a> (3) Profile Attribute: Avatar
+
+The avatar attribute is optional. It enables a domain owner to provide an avatar for the domain. The expected value is a fully qualified URL that must start with http:// or https:// that points to an image file.
```

Comment:
I'm fine with this, but thoughts on mandatory https? Can't think of a reason why any website would be http-only, and as a standard, might be better to encourage secure sites.